qemu/tests/multi_nics_stress.py

```diff
@@ -82,8 +83,8 @@ def run(test, params, env):
             server_ctl = server_vm.wait_for_login(timeout=login_timeout)
             error_context.context("Stop fireware on netperf server guest.",
                                   logging.info)
-            server_ctl.cmd("service iptables stop; iptables -F",
-                           ignore_all_errors=True)
+            if disable_firewall:
+                server_ctl.cmd(disable_firewall, ignore_all_errors=True)
```

Comment:
If you use params["disable_firewall"] above, here there is no need to judge. Because it will fail if the param is not given. And it should be fail, because we need to disable firewall.

Reply:
Update please help review.